plonk-core/src/constraint_system/ecc/curve_addition/variable_base_gate.rs

```diff
@@ -147,10 +147,10 @@ mod test {
                 .add(E::Fr::one(), E::Fr::one())
         });
 
-        // y1y2 - a * x1x2 (a=-1) => y1y2 + x1x2
+        // y1y2 - a * x1x2
         let y_numerator = composer.arithmetic_gate(|gate| {
             gate.witness(y1_y2, x1_x2, None)
-                .add(E::Fr::one(), E::Fr::one())
+                .add(E::Fr::one(), -P::COEFF_A)
```

Comment:
I would not negate it. The `COEFF_A` should containing the sign info with it no? We should not assume `a=-1` always.

Reply:
The `-` is from the operation itself. It is a subtraction so I use `-a`

Reply:
Ohhh my bad sorry...